__init__.py

```diff
@@ -622,9 +679,17 @@ def handle_no_backend_selected(self, message):
                     callback=handle_intent_aborted)
     def handle_stt_menu(self):
         self.state = SetupState.SELECTING_STT
-        self.gui["offlineSTT"] = self.setup.offline_stt_module
-        self.gui["onlineSTT"] = self.setup.online_stt_module
-        self.handle_display_manager("BackendLocalSTT")
+        model = get_stt_lang_configs(lang=self.selected_language, include_dialects=True)
+        supported_stt_engines = list()
+        for engine, configs in model.items():
+            # For Display purposes, we want to show the engine name without the underscore or dash and capitalized all
+            plugin_display_name = engine.replace("_", " ").replace("-", " ").title()
+            supported_stt_engines.append({"plugin_name": plugin_display_name,
+                                          "offline": configs[0]["offline"],
```

Comment:
this doesnt look correct, why are you selecting the first config option only? each plugin can provide multiple valid configs that should display here, for example this allows overriding lang model in STT engines for better dialect support

you should append every config as a new entry

Reply:
fixed